python/job_runner/runner.py

```diff
@@ -0,0 +1,87 @@
+import json
+import os
+
+from job_runner.job import Job
+from job_runner.reporting.message import Init, Finish
+
+
+class JobRunner(object):
+
+    def __init__(self, jobs_file="jobs.json"):
+        try:
+            with open(jobs_file, "r") as json_file:
+                self.jobs_data = json.load(json_file)
```

Comment:
Does not have to be object attribute as it is not used anywhere else outside this scope

Reply:
It is used in the `File` reporter: https://github.com/mortalisk/libres/blob/refactor_job_dispatch/python/job_runner/reporting/file.py#L134

Reply:
Then there is a typo: jobs_data  != job_data

Reply:
Correction this is a runner object, not a job object...